src/coreclr/src/jit/hwintrinsic.h

```diff
@@ -123,12 +126,14 @@ struct HWIntrinsicInfo
     NamedIntrinsic         id;
     const char*            name;
     CORINFO_InstructionSet isa;
-    int                    ival;
-    unsigned               simdSize;
-    int                    numArgs;
-    instruction            ins[10];
-    HWIntrinsicCategory    category;
-    HWIntrinsicFlag        flags;
+#ifdef TARGET_XARCH
+    int ival;
```

Comment:
There are actually relatively few of this one on x86 as well. It might be beneficial to have it be a helper method instead.
I might take a look at doing that after this goes in.